ExportData.cs

```diff
@@ -20,39 +20,27 @@ namespace Tracker
 {
     public static class ExportData
     {
-        private static string user;
-        private static string computerName;
-        private static string geo_x;
-        private static string geo_y;
-        private static string dynamoVersion;
-        private static string revitVersion;
-        private static string ip;
-        private static string filename;
+        internal static string user;
+        internal static string computerName;
+        internal static string dynamoVersion;
+        internal static string revitVersion;
+        internal static string ip;
+        internal static string latlng;
+        internal static string city;
+        internal static string country;
+        internal static string filename;
         private static string date;
 
-        internal static IList<IList<object>> Export(string filename, string dynamoversion)
+        internal static IList<IList<object>> Export()
```

Comment:
the purpose of this method seems to be : `create` an object that has all the data needed for export.
To make this more idiomatic C#, I would suggest to make a class `public class ExportData` with all the fields enumerated above (`user`, `computer`, `name`, `ip`, `etc`). The constructor of the class would initialise these fields with the correct values from the `Environment` class or by fetching IP etc.

Reply:
Agreed, this has mostly been resolved with the creation of the Data.Collect() method https://github.com/teamtreedyn/Binoculars/blob/master/Data.cs#L35

Although I opted for a static class. 

Discussion on this can continue as issue https://github.com/teamtreedyn/Binoculars/issues/18 therefore I'm marking this resolved.